server/container_create.go

```diff
@@ -1363,6 +1362,8 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	container.SetIDMappings(containerIDMappings)
 	if s.defaultIDMappings != nil && !s.defaultIDMappings.Empty() {
 		userNsPath := sb.UserNsPath()
+		rootPair := s.defaultIDMappings.RootPair()
```

Comment:
Do we already have a check somewhere that RootPair exists?

Reply:
Else we could just fail that at validation time while reading the configuration.